inc/change.class.php

```diff
@@ -159,6 +164,31 @@ function canCreateItem() {
    }
 
 
+   /**
+    * Get the ITIL object closed, solved or waiting status list
+    *
+    * @since 9.4.0
+    *
+    * @return an array
+   **/
+   static function getReopenableStatusArray() {
```

Comment:
This method is exactly the same for Changes, Problems and Tickets; so it must be moved to CommonITILObject super class